packages/grafana-data/src/types/datasource.ts

```diff
@@ -635,6 +640,7 @@ export interface MetricFindValue {
   value?: string | number;
   group?: string;
   expandable?: boolean;
+  nonApplicable?: boolean;
```

Comment:
Agree with the suggestion made in the design doc - we don't need to use `MetricFindValue`, returning an array of non-applicable keys id's should be enough.